src/main/java/frc/robot/subsystems/NoteVision.java

```diff
@@ -111,6 +125,24 @@ public List<Pose2d> getNotes(Pose2d pose) {
         return poses;
     }
 
+    public boolean checkForNote(DriveTrain driveTrain, int wantedNote, NoteVision noteVision) {
```

Comment:
You can also just pass in the robot pose. Again, no need to add a dependency on Drivetrain.